### PR TITLE
update images

### PR DESCRIPTION
**Pull request by yotamazriel**
- Update images
- Update tests after minio proxy size change

### CODE REVIEW

File: __snapshots__/main.test.ts.snap [hunk 1]
```diff
@@ -1251,7 +1251,7 @@ kind: Ingress
 metadata:
   annotations:
     kubernetes.io/ingress.class: public
-    nginx.ingress.kubernetes.io/proxy-body-size: 100m
+    nginx.ingress.kubernetes.io/proxy-body-size: 900m
```
Comment (arikmaor) [inline __snapshots__/main.test.ts.snap:1254]: Why did it change?

Comment (yotamazriel) [reply __snapshots__/main.test.ts.snap:1254]: The models KLA uploading are bigger than 100m

Comment (arikmaor) [reply __snapshots__/main.test.ts.snap:1254]: You've changes this manually?

Comment (yotamazriel) [reply __snapshots__/main.test.ts.snap:1254]: the pr that changing this value merged a few days ago here I just updating the tests

**Review by arikmaor — APPROVED** [review]
LGTM

**Pull request merged**